.github/stale.yml

```diff
@@ -0,0 +1,33 @@
+# Configuration for https://github.com/marketplace/stale
+
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 180
+
+# Number of days of inactivity before an Issue or Pull Request with the stale label is closed.
+# Set to false to disable. If disabled, issues still need to be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Ignore issues with an assignee (defaults to false)
+exemptAssignees: true
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  To help us better see what issues are still affecting our users, this issue
+  has been automatically marked as stale. If you still have this issue with an
+  up-to-date version of Certbot and are interested in seeing it resolved,
+  please add a comment letting us know.
```

Comment:
We should probably tell them that it'll be closed if they don't -- something like "If there is no further activity, this issue will be closed."

Reply:
I made the change with the same wording as you suggested except I added "automatically".

I initially left it out because I was worried about people adding a comment just to keep the issue open without actually checking if it's still a problem, but adding it probably prevents other problems by better explaining upfront what is going on.

Reply:
A valid worry, but yeah, I expect being transparent will help.